python/ray/train/lightning/lightning_trainer.py

```diff
@@ -427,6 +447,17 @@ def _check_checkpoint_configs(
                 "through `LightningConfigBuilder.checkpointing()`."
             )
 
+        # Auto-fill the AIR CheckpointConfig if the user didn't specify it.
+        if air_ckpt_config == CheckpointConfig():
+            save_top_k = ptl_ckpt_config.get("save_top_k", 1)
```

Comment:
We now default AIR to save 1 checkpoint when using it with lightning trainer -- is this intentional? PTL only saves 1 checkpoint by default?

Reply:
Yeah, in PTL,  the default value of `save_top_k` is 1.  Also, by default `monitor` is None which saves a checkpoint only for the last epoch.